src/main/java/mapper/UserMapper.java

```diff
@@ -0,0 +1,25 @@
+package mapper;
+
+import dto.user.UserLoginRequestDto;
+import dto.user.UserRegistrationRequestDto;
+import dto.user.UserResponseDto;
+import model.User;
+import org.mapstruct.BeanMapping;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+import org.mapstruct.NullValuePropertyMappingStrategy;
+
+@Mapper(componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl")
+public interface UserMapper {
+    UserResponseDto toDto(User user);
+
+    User toUser(UserRegistrationRequestDto requestDto);
+
+    @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
+    void updateUser(UserLoginRequestDto updatedRequestDto, @MappingTarget User bookToUpdate);
```

Comment:
```suggestion
    void updateUser(UserLoginRequestDto updatedRequestDto, @MappingTarget User userToUpdate);
```
be attentive with copy-pasting))